distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -434,24 +438,95 @@ public void testConfigurationFileWithInvalidFormat() throws IOException {
                 Arrays.asList(exportedFiles), is(not(empty())));
+        FileUtils.deleteDirectory(new File(getExportDirectory()));
     }
 
     @Test
     public void testExportOnTopOfFile() throws ConfigurationFileException, IOException {
-        String exportedDirectory = String.format("%s/etc/exported", ddfHome);
-        File file = new File(exportedDirectory);
+        File file = new File(getExportDirectory());
         file.createNewFile();
-        assertThat(String.format("Should not have been able to export to %s.", exportedDirectory),
-                console.runCommand(EXPORT_COMMAND), containsString(
-                        String.format("Failed to export all configurations to %s",
-                                exportedDirectory)));
+        String response = console.runCommand(EXPORT_COMMAND);
+        assertThat(String.format("Should not have been able to export to %s.", getExportDirectory()),
+                response, containsString(String.format("Failed to export all configurations to %s",
+                        getExportDirectory())));
         file.delete();
     }
+
+    /*
+        Different Scenarios we want to test.
+        1.  testExport --> should check to see if there are the correct files
+        2.  testExportOnTopOfFile --> should check to see if the correct error is thrown
+        3.  testExport after saving a new configuration --> check to see if it is included in the export
+        4.  testExport after deleting a configuration --> check to make sure it isn't included in the export
+        5.  testImport after exporting?
+        6.
+     */
+
+    @Test
+    public void textExportAfterSavingAConfiguration() throws Exception {
+        managedServiceNewConfig1.addConfigurationFileAndWait(configAdmin);
+        console.runCommand(EXPORT_COMMAND);
+
+        assertThat("Saved configuration should be exported.",
+                new File(getPathToExportedConfig(managedServiceNewConfig1.pid)).isFile(),
+                is(true));
+        FileUtils.deleteDirectory(new File(getExportDirectory()));
+    }
+
+    @Test
+    public void testExportAfterDeletingAConfiguration() throws Exception {
+        managedServiceNewConfig2.addConfigurationFileAndWait(configAdmin);
+        configAdmin.getConfiguration(managedServiceNewConfig2.pid, null).delete();
+        console.runCommand(EXPORT_COMMAND);
+
+        assertThat("Deleted configuration should not be exported.",
+                new File(getPathToExportedConfig(managedServiceNewConfig2.pid)).isFile(),
+                is(false));
+        FileUtils.deleteDirectory(new File(getExportDirectory()));
+    }
+
+    @Test
+    public void testExportSystemPropertiesFile() throws Exception {
+        File systemProperties = new File(getExportDirectory() + "/etc/system.properties");
+
+        // clean any previous exports
+        if (systemProperties.exists()) {
+            systemProperties.delete();
```

Comment:
It may not. in the next commit, this logic was swapped out for the deleteDirectory method the other methods used.